packages/fiori-freestyle-writer/templates/basic/add/webapp/controller/App.controller.ts

```diff
@@ -5,6 +5,7 @@ import Controller from "sap/ui/core/mvc/Controller";
  */
 export default class App extends Controller {
 
+    /*eslint-disable @typescript-eslint/no-empty-function*/
```

Comment:
Would this still be an issue if the function contains a comment maybe explaining what the `onInit` function is and when it is called?
In addition, wouldn't it be best practice to also call `super.onInit()`?

Reply:
Changing to call `super.onInit();` and the app no longer starts. I think this is different to `super.init();` in the init() function.
```
    public onInit(): void {
        super.onInit();
    }
```

Reply:
Sorry, yes, you are correct, the `onInit` isn't an overridden function.